packages/react/src/get-label-from-stack-trace.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+export const getFunctionNameFromStackTraceLine = (line: string): ?string => {
+  // V8
+  let match = /^\s+at\s+([A-Za-z0-9$.]+)\s/.exec(line)
+
+  if (match) {
+    // The match may be something like 'Object.createEmotionProps'
+    const parts = match[1].split('.')
+    return parts[parts.length - 1]
+  }
+
+  // Safari / Firefox
+  match = /^([A-Za-z0-9$.]+)@/.exec(line)
+  if (match) return match[1]
+
+  return undefined
+}
+
+const internalReactFunctionNames = new Set([
```

Comment:
```suggestion
const internalReactFunctionNames = /* #__PURE__ */ new Set([
```

Do you think minifiers will be able to remove all this code @Andarist? Given they're all just declaring functions, I would imagine it should be fine?

Reply:
IIRC this should be removed if we add the PURE comment here, the passed argument is pure so that part is fine - it's only this constructor call that is "dangerous". We already add the PURE annotation in very similar place, like here:
https://github.com/emotion-js/emotion/blob/2bac69b6c058d007cfc190e94490c0cb225ca40c/packages/cache/src/stylis-plugins.js#L86

And the rest is totally fine as those are just function declarations/expressions so they should be easily removed if they stay unreferenced.